packages/knip/src/plugins/ladle/index.ts

```diff
@@ -0,0 +1,43 @@
+import type { EnablerPatterns } from '#p/types/config.js';
+import type { IsPluginEnabled, Plugin, ResolveConfig, ResolveEntryPaths } from '#p/types/plugins.js';
+import { hasDependency, load } from '#p/util/plugin.js';
+import type { LadleConfig } from './types.js';
+import { toEntryPattern } from '../../util/protocols.js';
+import { resolveConfig as resolveVitestConfig } from '../vitest/index.js';
+
+// https://ladle.dev/docs/config
+
+const title = 'ladle';
+
+const enablers: EnablerPatterns = [/^@ladle\//];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) => hasDependency(dependencies, enablers);
+
+const config = ['.ladle/config.{mjs,js,ts}'];
+
+const stories: string[] = ['src/**/*.stories.{js,jsx,ts,tsx,mdx}'];
+const restEntry: string[] = ['.ladle/components.{js,jsx,ts,tsx}'];
+const entry: string[] = [...restEntry, ...stories];
+
+const project = ['.ladle/**/*.{js,jsx,ts,tsx}'];
+
+const resolveEntryPaths: ResolveEntryPaths<LadleConfig> = async localConfig => {
+  const localStories = typeof localConfig.stories === 'string' ? [localConfig.stories] : localConfig.stories;
+  const patterns = [...restEntry, ...(localStories ?? stories)];
+
+  return patterns.map(toEntryPattern);
+};
+
+const resolveConfig: ResolveConfig<LadleConfig> = async (localConfig, options) =>
+  localConfig.viteConfig ? resolveVitestConfig(await load(localConfig.viteConfig), options) : [];
```

Comment:
Please expand it a bit and use something like `join(options.configFileDir, localConfig.viteConfig)` so that it is resolved relative the Ladle config file dir.

Reply:
Oh never mind, now I read https://ladle.dev/docs/config/#viteconfig and see that the path should be absolute anyway (wasn't clear from the fixture).

Reply:
Not sure, just the example uses an absolute path, I could not find something in the documentation stating that is has to be absolute.

I think it would be great to support both, so I now check if the path is already absolute and if not, I join it with `options.cwd`. (I did not check before what's inside options, so helpful 🤣 ).

At least on my machine with our config it now works like a charm :)